tests/formats/pdb.cpp

```diff
@@ -343,17 +344,19 @@ TEST_CASE("PDB files with big values") {
     }
 
     SECTION("CONNECT with too many atoms") {
-        auto tmpfile = NamedTempPath(".pdb");
-
-        auto frame = Frame();
-        for(size_t i=0; i<110000; i++) {
-            frame.add_atom(Atom("A"), {0.0, 0.0, 0.0});
+        if (!is_valgrind_and_travis()) {
```

Comment:
Or maybe this reduced the timings!

The same should be done with the GRO format to ensure that test finishes in time as well.

Reply:
Will do! I forgot about GRO

Reply:
Done and rebased!

Reply:
Great!